xarray/tests/test_dataarray.py

```diff
@@ -2519,6 +2519,69 @@ def test_combine_first(self):
                              [('x', ['a', 'b', 'd']), ('y', [-1, 0])])
         self.assertDataArrayEqual(actual, expected)
 
+    def test_sortby(self):
```

Comment:
To be honest, I would say the entire `test_dataarray::test_sortby` is redundant, because it's essentially a carbon copy of `test_dataset::test_sortby`.  I've gotten rid of a few lines in the latest push, but could trim more if you like.